### DISCUSSION TITLE
MQTT Update

### DISCUSSION THREAD

**Discussion by cj82rnk** [arendst/Tasmota#22192] [Q&A]
Sorry, ich kann leider kein Englisch, vielleicht kann mir ja trotzdem jemand helfen.
Ich habe Tasmota in meinem ioBroker eingebunden. Dort werden die Objekte bei jeder Änderung von Spannung, Strom usw. sofort gepostet. Diese Werte versuche ich nun über MQTT an anderer Stelle als den ioBroker abzurufen.
Ich bekomme aber bspw. bei stat/xxx/SENSOR.ENERGY.Power keinen Wert. Diesen Pfad gibt es aber so im ioBroker.
Das schalten mit stat/xxx/POWER funktioniert dagegen.

Ich komme mit tele/xxx/SENSOR zwar an meine Werte, die werden von Tasmota aber nur jede Minute aktualisiert, der ioBroker bekommt das aber sofort.
Was mache ich falsch? Über welchen Pfad bekomme ich denn auch die Änderungen mitgeteilt ??

**Comment by sfromis** [comment]
Command `PowerDelta`
